bundles/org.openhab.binding.energidataservice/src/main/java/org/openhab/binding/energidataservice/internal/ApiController.java

```diff
@@ -198,7 +199,7 @@ public Collection<DatahubPricelistRecord> getDatahubPriceLists(GlobalLocationNum
         }
 
         Request request = httpClient.newRequest(ENDPOINT + DATASET_PATH + DATASET_NAME_DATAHUB_PRICELIST)
-                .param("filter", mapToFilter(filterMap)) //
+                .timeout(REQUEST_TIMEOUT_MS, TimeUnit.MILLISECONDS).param("filter", mapToFilter(filterMap)) //
```

Comment:
Formatting and unit commented above.
```suggestion
                .timeout(REQUEST_TIMEOUT_SECONDS, TimeUnit.SECONDS) //
                .param("filter", mapToFilter(filterMap)) //
```

Reply:
Done